### ISSUE TITLE
AttributeError: 'Namespace' object has no attribute 'fileArgs'

### ISSUE THREAD

**Issue by KyleKing** [TeamMsgExtractor/msg-extractor#335]
**Bug Metadata**

* Version of extract_msg: [0.38.4 and 0.39.0, but not 0.30.11]
* Your python version: Python [3.10.7]
* How did you launch extract_msg?
  - [x] My command line

**Describe the bug**

Parser error for positional arguments to `extract_msg ... --out-name ...`

I believe the below line should be: `if options.outName and options.msgs and len(options.msgs) > 0:`

https://github.com/TeamMsgExtractor/msg-extractor/blob/ab7618c4da4a4090ddb09e8246af179baaa29e03/extract_msg/utils.py#L414

Because of this declaration:

https://github.com/TeamMsgExtractor/msg-extractor/blob/ab7618c4da4a4090ddb09e8246af179baaa29e03/extract_msg/utils.py#L360-L362

**Traceback**

```sh
Traceback (most recent call last):
  File "../bin/extract_msg", line 8, in <module>
    sys.exit(main())
  File "../venv/lib/python3.10/site-packages/extract_msg/__main__.py", line 14, in main
    args = utils.getCommandArgs(sys.argv[1:])
  File "../venv/lib/python3.10/site-packages/extract_msg/utils.py", line 402, in getCommandArgs
    if options.outName and options.fileArgs and len(options.fileArgs) > 0:
AttributeError: 'Namespace' object has no attribute 'fileArgs'
```

**Comment by TheElementalOfDestruction** [comment]
Tracked back to version 0.34.0, no idea how it's been around that long without being caught by users or my own testing 😅 

Pretty sure that was meant to be .msgs or whatever instead of .fileArgs, so I'll fix that in the next release with a few other things

**Comment by KyleKing** [comment]
Thanks! Haha, we had pinned `extract-msg` to 0.30.11 😅

**Comment by TheElementalOfDestruction** [comment]
This has been fixed in version 0.39.1
